description: disable upstream tests that need data to be generated
author: Michael Gilbert <mgilbert@debian.org>

--- a/configure.ac
+++ b/configure.ac
@@ -2875,7 +2875,6 @@ WINE_CONFIG_MAKEFILE(dlls/gameux/tests)
 WINE_CONFIG_MAKEFILE(dlls/gamingtcui)
 WINE_CONFIG_MAKEFILE(dlls/gdi.exe16,enable_win16)
 WINE_CONFIG_MAKEFILE(dlls/gdi32)
-WINE_CONFIG_MAKEFILE(dlls/gdi32/tests)
 WINE_CONFIG_MAKEFILE(dlls/gdiplus)
 WINE_CONFIG_MAKEFILE(dlls/gdiplus/tests)
 WINE_CONFIG_MAKEFILE(dlls/glu32)
@@ -2922,7 +2921,6 @@ WINE_CONFIG_MAKEFILE(dlls/iprop)
 WINE_CONFIG_MAKEFILE(dlls/irprops.cpl)
 WINE_CONFIG_MAKEFILE(dlls/itircl)
 WINE_CONFIG_MAKEFILE(dlls/itss)
-WINE_CONFIG_MAKEFILE(dlls/itss/tests)
 WINE_CONFIG_MAKEFILE(dlls/joy.cpl)
 WINE_CONFIG_MAKEFILE(dlls/jscript)
 WINE_CONFIG_MAKEFILE(dlls/jscript/tests)
@@ -2930,7 +2928,6 @@ WINE_CONFIG_MAKEFILE(dlls/jsproxy)
 WINE_CONFIG_MAKEFILE(dlls/jsproxy/tests)
 WINE_CONFIG_MAKEFILE(dlls/kerberos)
 WINE_CONFIG_MAKEFILE(dlls/kernel32)
-WINE_CONFIG_MAKEFILE(dlls/kernel32/tests)
 WINE_CONFIG_MAKEFILE(dlls/kernelbase)
 WINE_CONFIG_MAKEFILE(dlls/kernelbase/tests)
 WINE_CONFIG_MAKEFILE(dlls/keyboard.drv16,enable_win16)
@@ -3118,7 +3115,6 @@ WINE_CONFIG_MAKEFILE(dlls/nsi)
 WINE_CONFIG_MAKEFILE(dlls/nsi/tests)
 WINE_CONFIG_MAKEFILE(dlls/nsiproxy.sys)
 WINE_CONFIG_MAKEFILE(dlls/ntdll)
-WINE_CONFIG_MAKEFILE(dlls/ntdll/tests)
 WINE_CONFIG_MAKEFILE(dlls/ntdsapi)
 WINE_CONFIG_MAKEFILE(dlls/ntdsapi/tests)
 WINE_CONFIG_MAKEFILE(dlls/ntoskrnl.exe)
@@ -3203,7 +3199,6 @@ WINE_CONFIG_MAKEFILE(dlls/riched20/tests
 WINE_CONFIG_MAKEFILE(dlls/riched32)
 WINE_CONFIG_MAKEFILE(dlls/riched32/tests)
 WINE_CONFIG_MAKEFILE(dlls/rpcrt4)
-WINE_CONFIG_MAKEFILE(dlls/rpcrt4/tests)
 WINE_CONFIG_MAKEFILE(dlls/rsabase)
 WINE_CONFIG_MAKEFILE(dlls/rsaenh)
 WINE_CONFIG_MAKEFILE(dlls/rsaenh/tests)
@@ -3246,10 +3241,8 @@ WINE_CONFIG_MAKEFILE(dlls/shdocvw)
 WINE_CONFIG_MAKEFILE(dlls/shdocvw/tests)
 WINE_CONFIG_MAKEFILE(dlls/shell.dll16,enable_win16)
 WINE_CONFIG_MAKEFILE(dlls/shell32)
-WINE_CONFIG_MAKEFILE(dlls/shell32/tests)
 WINE_CONFIG_MAKEFILE(dlls/shfolder)
 WINE_CONFIG_MAKEFILE(dlls/shlwapi)
-WINE_CONFIG_MAKEFILE(dlls/shlwapi/tests)
 WINE_CONFIG_MAKEFILE(dlls/slbcsp)
 WINE_CONFIG_MAKEFILE(dlls/slc)
 WINE_CONFIG_MAKEFILE(dlls/slc/tests)
@@ -3301,11 +3294,9 @@ WINE_CONFIG_MAKEFILE(dlls/unicows)
 WINE_CONFIG_MAKEFILE(dlls/updspapi)
 WINE_CONFIG_MAKEFILE(dlls/url)
 WINE_CONFIG_MAKEFILE(dlls/urlmon)
-WINE_CONFIG_MAKEFILE(dlls/urlmon/tests)
 WINE_CONFIG_MAKEFILE(dlls/usbd.sys)
 WINE_CONFIG_MAKEFILE(dlls/user.exe16,enable_win16)
 WINE_CONFIG_MAKEFILE(dlls/user32)
-WINE_CONFIG_MAKEFILE(dlls/user32/tests)
 WINE_CONFIG_MAKEFILE(dlls/userenv)
 WINE_CONFIG_MAKEFILE(dlls/userenv/tests)
 WINE_CONFIG_MAKEFILE(dlls/usp10)
@@ -3401,9 +3392,7 @@ WINE_CONFIG_MAKEFILE(dlls/wing32)
 WINE_CONFIG_MAKEFILE(dlls/winhttp)
 WINE_CONFIG_MAKEFILE(dlls/winhttp/tests)
 WINE_CONFIG_MAKEFILE(dlls/wininet)
-WINE_CONFIG_MAKEFILE(dlls/wininet/tests)
 WINE_CONFIG_MAKEFILE(dlls/winmm)
-WINE_CONFIG_MAKEFILE(dlls/winmm/tests)
 WINE_CONFIG_MAKEFILE(dlls/winnls.dll16,enable_win16)
 WINE_CONFIG_MAKEFILE(dlls/winnls32)
 WINE_CONFIG_MAKEFILE(dlls/winscard)
